src/index.ts

```diff
@@ -56,24 +66,59 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
-function addComment(crawlerId: string): void {
-  try {
-    const pathArray = CRAWLER_API_BASE_URL.split('/');
-    const protocol = pathArray[0];
-    const host = pathArray[2];
-    const baseUrl = `${protocol}//${host}`;
+function findCommentPredicate(crawlerId: string, comment: Comment): boolean {
+  return comment.body ? comment.body.includes(crawlerId) : true;
+}
 
-    const message = `<p>Check your created <a href="${baseUrl}/admin/crawlers/${crawlerId}/overview" target="_blank">Crawler</a></p>
-    <p>Check your created index on your <a href="https://www.algolia.com/apps/${ALGOLIA_APP_ID}/explorer/browse/${CRAWLER_NAME}" target="_blank">Algolia Application</a></p>`;
+async function findComment(
+  prNumber: number,
+  crawlerId: string
+): Promise<Comment | undefined> {
+  const parameters = {
+    owner: github.context.repo.owner,
+    repo: github.context.repo.repo,
+    issue_number: prNumber,
+  };
 
+  for await (const { data: comments } of octokit.paginate.iterator(
+    octokit.rest.issues.listComments,
+    parameters
+  )) {
+    // Search each page for the comment
+    const gaComment = comments.find((comment) =>
+      findCommentPredicate(crawlerId, comment)
```

Comment:
You will find every comment with the crawlerId which might be a comment from someone else, can you filter by author?